src/effects/VST/VSTEffect.h

```diff
@@ -264,19 +417,17 @@ class VSTEffect final
    int GetMidiInCount() const override;
    int GetMidiOutCount() const override;
 
-   sampleCount GetLatency() const override;
-
    size_t SetBlockSize(size_t maxBlockSize) override;
    size_t GetBlockSize() const override;
 
    bool IsReady();
-   bool ProcessInitialize(EffectSettings &settings, double sampleRate,
-      sampleCount totalLen, ChannelNames chanMap) override;
-   bool DoProcessInitialize(double sampleRate);
-   bool ProcessFinalize() override;
-   size_t ProcessBlock(EffectSettings &settings,
-      const float *const *inBlock, float *const *outBlock, size_t blockLen)
-      override;
+
+   bool Process(EffectInstance& instance, EffectSettings& settings)
```

Comment:
This function should be marked `override` and its definition should be non-inline

Reply:
ok, done